modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -221,23 +243,22 @@ func (suite *KeeperTestSuite) TestDistributeTimeoutFee() {
 
 	err := suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), identifiedPacketFee)
 	suite.Require().NoError(err)
+	// escrow a second packet fee to test with multiple fees distributed
+	err = suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), identifiedPacketFee)
```

Comment:
I'd like to see the escrow at least with a different refund address so we can assert that the refunding logic works correctly

Reply:
We check below that the amount has been correctly refunded but I can do this.